src/openrct2/actions/ParkSetEntranceFeeAction.cpp

```diff
@@ -42,17 +42,25 @@ void ParkSetEntranceFeeAction::Serialise(DataSerialiser& stream)
 
 GameActions::Result ParkSetEntranceFeeAction::Query() const
 {
-    bool noMoney = (GetGameState().ParkFlags & PARK_FLAGS_NO_MONEY) != 0;
-    bool forceFreeEntry = !ParkEntranceFeeUnlocked();
-    if (noMoney || forceFreeEntry)
+    if ((GetGameState().ParkFlags & PARK_FLAGS_NO_MONEY) != 0)
```

Comment:
keep the assignments as the variables provided a bit better documentation, like:
```c++
	    if (bool noMoney = (GetGameState().ParkFlags & PARK_FLAGS_NO_MONEY) != 0))
```

Reply:
Neat idea, done